libcloud/compute/drivers/gce.py

```diff
@@ -2243,7 +2251,14 @@ def ex_create_image(self, name, volume, description=None, family=None,
             image_data['rawDisk'] = {'source': volume, 'containerType': 'TAR'}
         else:
             raise ValueError('Source must be instance of StorageVolume or URI')
-
+        if guest_os_features:
+            image_data['guestOsFeatures'] = []
+            for feature in guest_os_features:
+                if feature in possble_features:
```

Comment:
that should be `for feature in possible_features:` and also showing this case needs a test :-)